client/src/modules/Registry/pages/Student/index.tsx

```diff
@@ -259,18 +310,32 @@ function sortByStartDate(a: Course, b: Course) {
   return a.startDate.localeCompare(b.startDate);
 }
 
-function isCourseOpenForRegistry(course: Course) {
+function enrolledOtherCourses(studentStats: StudentStats[] | undefined, courses: Course[]) {
+  return (
+    studentStats?.reduce((acc, { isExpelled, isCourseCompleted, courseId }) => {
+      const course = courses?.find(el => el.id === courseId);
+      if (!isExpelled && !isCourseCompleted && course && !course?.completed) {
```

Comment:
`course && !course?.completed` 

you can replace it with `course?.completed === false`

Reply:
done